src/shared/ui/Items/Time/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { DesktopTimePicker } from "@mui/x-date-pickers/DesktopTimePicker"
+
+type Props = {
+  label?: string
+  value?: string
+  onChange: (value: Date | null) => void
+}
+
+export const TimeItemBase = ({ label, value, onChange }: Props) => {
+  const formatedValue = value ? new Date(value) : null
+
+  return <DesktopTimePicker<Date> label={label} value={formatedValue} onChange={onChange} />
```

Comment:
Can we really use generics in JSX?

Reply:
Yes, we can. 
I have investigated how date and time picker components work under the hood. And yes, we can provide a type of value via generic.
The default value type is a string but for my purposes, the type Date is more comfortable